src/main/java/ca/on/oicr/pineryreports/reports/impl/LocationMissingReport.java

```diff
@@ -0,0 +1,174 @@
+package ca.on.oicr.pineryreports.reports.impl;
+
+import static ca.on.oicr.pineryreports.util.SampleUtils.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Predicate;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.ParseException;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+
+import ca.on.oicr.pinery.client.HttpResponseException;
+import ca.on.oicr.pinery.client.PineryClient;
+import ca.on.oicr.pinery.client.SampleClient.SamplesFilter;
+import ca.on.oicr.pineryreports.data.ColumnDefinition;
+import ca.on.oicr.pineryreports.reports.TableReport;
+import ca.on.oicr.pineryreports.util.CommonOptions;
+import ca.on.oicr.ws.dto.SampleDto;
+
+public class LocationMissingReport extends TableReport {
+
+  public static final String REPORT_NAME = "location-missing";
+  private List<SampleDto> locationMissing = new ArrayList<>();
+
+  private static final Option OPT_AFTER = CommonOptions.after(false);
+  private static final Option OPT_BEFORE = CommonOptions.before(false);
+  private static final Option OPT_PROJECT = CommonOptions.project(true);
```

Comment:
Project is declared as required, but the report seems to handle lack of project specification?

Reply:
Right, I think I changed my mind partway through on this. Will update.